api/mysagw/healthz/health_checks.py

```diff
@@ -34,7 +34,7 @@ def _check_pending_migrations(db_name):
 def _check_models(database):
     """Check model fetching on database."""
     # Retrieve object
-    identity = Identity.objects.get(email="sagw@sagw.ch", is_organisation=True)
+    identity = Identity.objects.first()
     assert identity
```

Comment:
You can leave it as this is an emergency PR, but asserts in production code are a bad idea, as they can get optimized away. I'd rather have something like

```python
if not Identity.objects.exists():
    return {database: {"ok": False}}